docs/modules/ROOT/pages/relay-api-reference.adoc

```diff
@@ -171,7 +171,7 @@ An example of the response:
 
 [[sign-endpoint]]
 == Sign Endpoint
-To sign arbitrary data with your Relay private key make a `POST` request to `/sign` with a payload containing the hex string to sign. The payload format is:
+To sign arbitrary messages according to the https://eips.ethereum.org/EIPS/eip-191[EIP-191 Standard] (prefixed by `\x19Ethereum Signed Message:\n`) with your Relay private key make a `POST` request to `/sign` with a payload containing the hex string to sign. The payload format is:
```

Comment:
Maybe link it [EIP-191 Standard](https://eips.ethereum.org/EIPS/eip-191) like so?